.github/workflows/check.yml

```
@@ -0,0 +1,20 @@
+name: check
```

```suggestion
name: format_and_lint
```
the name should be descriptive. I've started using a `pre-commit` workflow to shame all who don't conform

```
name: pre-commit-check
on:
  push:
    branches:
      - master
  pull_request:

jobs:
  build:
    runs-on: macOS-latest
    steps:
    - uses: actions/checkout@v1

    - name: Install prerequisites
      run: |
        brew install tfenv tflint terraform-docs pre-commit
        pre-commit install
        tfenv install
    - name: pre-commit run all
      run: |
        pre-commit run -a
```